### PR TITLE
add docs for letsencrypt SSL cert with nginx proxy

### PR DESCRIPTION
**Pull request by VikramTiwari**
For https://github.com/gdg-x/hub/issues/98

### CODE REVIEW

File: docs/nginx-setup.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Setting up LetsEncrypt Certificate with Nginx Reverse Proxy
+
+- setup virtual host config as in `./nginx.conf`
+
+- create a softlink
+```
+  sudo ln -s /etc/nginx/sites-available/subsites /etc/nginx/sites-enabled/subsites
+```
+
+- get a cert using LetsEncrypt (Make sure LetsEncrypt is installed)
```
Comment (Splaktar) [inline docs/nginx-setup.md]: Looks really good! Can you please add a link to instructions for installing Let's Encrypt? I know that there are various different NodeJS and platform specific packages/libs.

File: docs/nginx-setup.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+- create SSL cert conf file for nginx
+```
+sudo nano /etc/nginx/snippets/ssl-hub.gdgx.io.conf
+
+## in the file
+ssl_certificate /etc/letsencrypt/live/hub.gdgx.io/fullchain.pem;
+ssl_certificate_key /etc/letsencrypt/live/hub.gdgx.io/privkey.pem;
+```
```
Comment (Splaktar) [inline docs/nginx-setup.md]: Can we split this into two blocks? One for the command one for the file contents.

File: docs/nginx-setup.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+- create SSL params conf for nginx
+```
+sudo nano /etc/nginx/snippets/ssl-params.conf
+
+## in the file
+# from https://cipherli.st/
+# and https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
```
Comment (Splaktar) [inline docs/nginx-setup.md]: Can we split this into two blocks? One for the command one for the file contents.

**Review by Splaktar — CHANGES_REQUESTED** [review]
Just some minor nitpicks, thank you!

**Comment by VikramTiwari** [comment]
@Splaktar Fixed and added some more docs.

**Comment by Splaktar** [comment]
Awesome, can you link to the letsencrypt doc from the nginx-setup doc please?

Once we get it merged, I'll link to the docs from the wiki.

**Comment by VikramTiwari** [comment]
@Splaktar Done! They should be auto-linked internally and you will need to create a link from wiki.

**Review by Splaktar — APPROVED** [review]

**Comment by Splaktar** [comment]
Thank you very much!

**Pull request merged**